### PR TITLE
Upping versions and fixing most dart compiler warnings.

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: web/core_animation.dart [hunk 1]
```diff
@@ -7,10 +7,8 @@
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 import 'dart:html';
-import 'dart:js';
 import 'package:polymer/polymer.dart';
-import 'package:core_elements/core_animation.dart';
```
Comment (jakemac53) [inline web/core_animation.dart]: This actually needs to stay, for the html import it provides.

**Comment by jakemac53** [comment]
just the one comment, otherwise lgtm

File: web/core_animation.dart (hunk 1 above)
Comment (domesticmouse) [inline web/core_animation.dart]: Added it back, with a note.

**Comment by jakemac53** [comment]
thanks!

**Pull request merged**